src/mailadm/cmdline.py

```diff
@@ -78,10 +101,25 @@ def setup_bot(ctx, email, password, show_ffi):
     if show_ffi:
         ac.add_account_plugin(FFIEventLogger(ac))
 
+    mailadmdb = get_mailadm_db(ctx)
+    with mailadmdb.read_connection() as rconn:
+        mail_domain = rconn.config.mail_domain
+        admingrpid_old = rconn.config.admingrpid
+
     if not ac.is_configured():
-        assert email and password, (
-            "you must specify --email and --password once to configure this database/account"
-        )
+        if email and not password:
+            if email.split("@")[1] == mail_domain:
+                print("--password not specified, creating account automatically... ")
+                email, password = create_bot_account(ctx, email)
+            else:
+                ctx.fail("You need to provide --password if you want to use an existing account "
```

Comment:
if you want to use an "outside" e-mail account for the admin bot.

Reply:
Yes, but the user already knows that they specified an outside e-mail address, so no need to tell them at this point. That it's existing is the more relevant info for them.